admin/src/components/generator/ContentGeneratorPanel.jsx

```diff
@@ -0,0 +1,483 @@
+						Prompt Template to choose
+					</div>
+
+					<div className="urlslab-content-gen-panel-control-item-selector">
+						<SingleSelectMenu
+							key={ selectedPromptTemplate }
+							items={ promptTemplateSelections }
+							name="context_menu"
+							defaultAccept
+							autoClose
+							defaultValue={ selectedPromptTemplate }
+							onChange={ handlePromptTemplateChange }
+						/>
+					</div>
+				</div>
+
+				<div className="urlslab-content-gen-panel-control-item">
+					<TextAreaEditable
+						liveUpdate
+						val={ promptVal }
+						defaultValue=""
+						label={ __( 'Prompt' ) }
+						rows={ 10 }
+						allowResize
+						onChange={ handlePromptChange }
+						required
+						placeholder={ contextTypePromptPlaceholder[ dataSource ] }
+						description={ __( 'Prompt to be used while generating content' ) } />
+				</div>
+
+				<div className="urlslab-content-gen-panel-control-item">
+					<SingleSelectMenu
+						key={ modelName }
+						items={ {
+							'gpt-3.5-turbo': 'OpenAI GPT 3.5 Turbo',
+							'gpt-4': 'OpenAI GPT 4',
+							'text-davinci-003': 'OpenAI GPT Davinci 003',
+						} }
+						name="mode_name_menu"
+						defaultAccept
+						autoClose
+						defaultValue={ modelName }
+						onChange={ ( val ) => setModelName( val ) }
+					/>
+				</div>
+
+				<div className="urlslab-content-gen-panel-control-item">
+					<Button active onClick={ handleGenerateContent }>
+						{
+							isGenerating ? ( <Loader /> ) : __( 'Generate Text' )
+						}
+					</Button>
+				</div>
+
+				<div className="urlslab-content-gen-panel-control-item">
+					{ errorGeneration && <div>{ errorGeneration }</div> }
+				</div>
+
+			</div>
+			<div className="urlslab-content-gen-panel-editor">
+				{
+					editorLoading && <Loader />
+				}
+				<TinyMCE
+					onInit={ ( evt, editor ) => {
+						editorRef.current = editor;
+						setEditorLoading( false );
+					} }
+					value={ editorVal }
+					onEditorChange={ ( input ) => setEditorVal( input ) }
+					init={ {
+						skin: false,
+						content_css: false,
+						height: '80vh',
+						menubar: false,
+						entity_encoding: 'raw',
+						plugins: [
+							'advlist', 'autolink', 'lists', 'link', 'image', 'anchor', 'media', 'table', 'code',
+						],
+						toolbar: [ 'blocks | bold italic forecolor | alignleft aligncenter',
+							'alignright alignjustify | bullist numlist outdent indent | code help' ],
+						content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:20px }',
+					} }
+				/>
+				{
+					Object.keys( postTypes ).length !== 0 && ! editorLoading && (
+						<div>
+							<SingleSelectMenu
+								key={ postType }
+								items={ postTypes }
+								name="post_type_menu"
+								defaultAccept
+								autoClose
+								defaultValue={ postType }
+								onChange={ ( val ) => setPostType( val ) }
+							/>
+
+							<Button onClick={ handleCreatePost }>Create Post</Button>
```

Comment:
missing localization functions

Reply:
@yasha-dev1 buttons stucked on each other will stay as is for now?

Reply:
Yes, after using the framework, we'll work on the UI